tests/unit/scripts/test_kg_build.py

```
@@ -409,8 +530,794 @@ def __str__(self) -> str:
+    sanitized = kg._sanitize_property_value(raw)
+    assert isinstance(sanitized, str)
+    parsed = json.loads(sanitized)
+    assert parsed["level1"]["level2"]["level3"]["level4"]["level5"] == "deep"
+    # Dict with numeric-like string keys
+    raw = {"1": "one", "2": "two", "10": "ten"}
+    sanitized = kg._sanitize_property_value(raw)
+    assert isinstance(sanitized, str)
+    parsed = json.loads(sanitized)
+    assert parsed["1"] == "one"
+    assert parsed["10"] == "ten"
+    repo_dir = tmp_path / "repo"
```

Ruff (0.13.2)

Undefined name `tmp_path`

(F821)